modular_skyrat/modules/customization/modules/surgery/organs/ipc.dm

```diff
@@ -69,12 +69,12 @@
 			owner.Dizzy(30)
```

Comment:
All the values in this proc should use the SECONDS macro as they indicate time. I know it's not your doing, but if you touch it, you can improve it.